source/gui/addonStoreGui/controls/storeDialog.py

```diff
@@ -334,20 +334,20 @@ def _toggleFilterControls(self):
 
 	def onListTabPageChange(self, evt: wx.EVT_CHOICE):
 		self.searchFilterCtrl.SetValue("")
-

```

Comment:
please restore the new lines to this function. They exist for readability to separate the function into sections